partner_medical_medication/views/res_partner_view.xml

```diff
@@ -12,23 +12,17 @@
         <field name="inherit_id" ref="base.view_partner_form"/>
         <field name="arch" type="xml">
             <xpath expr="//div[@name='button_box']" position="inside">
-                <field name="patient_ids" invisible="True"/>
-                <button  
-                    name="partner_medical_medication.partner_action_medication" type="action" 
-                    class="oe_stat_button" icon="fa-list"
-                    attrs="{'invisible': [('patient_ids', '=', [])]}">
-                    <span>Medications</span>
-                </button>
+                <field name="medication_ids" invisible="True" />
+                <button name="medical_medication.medical_patient_medication_action"
+                        string="Medications"
+                        type="action"
+                        class="oe_stat_button"
+                        icon="fa-list"
+                        context="{'search_default_id': medication_ids}"
+                        attrs="{'invisible': [('medication_ids', '=', [])]}"
```

Comment:
Do we need this `attrs="{'invisible':`, it is also a meaningful information to know that a patient never add medications.